pkgs/development/python-modules/responses/default.nix

```
@@ -17,7 +17,7 @@
 
 buildPythonPackage rec {
   pname = "responses";
-  version = "0.23.3";
+  version = "0.24.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/getsentry/responses/blob/5579f8ecbe4bc59e04bc334f98c8a68b93c57cb3/setup.py#L79

---

Thanks